stonesoup/updater/chernoff.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from ..base import Property
+from .base import Updater
+from ..types.prediction import GaussianMeasurementPrediction
+from ..types.update import GaussianStateUpdate
```

Comment:
See suggestions below which need this.
```suggestion
from ..types.prediction import MeasurementPrediction
from ..types.update import Update
```